CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
    * CHANGED: Microoptimisation in EdgeInfo. [#5733](https://github.com/valhalla/valhalla/pull/5733)
    * CHANGED: Merge cost and tyr inline tests into single executable [#5735](https://github.com/valhalla/valhalla/pull/5735) 
    * ADDED: Add option `edge.hov_type` to trace attributes [#5741](https://github.com/valhalla/valhalla/pull/5741)
+   * ADDED: Bicycle improvements. use_roads updates and adds support for separate, cycleway:right:oneway, cycleway:left:oneway, and cycleway:both [#5750](https://github.com/valhalla/valhalla/pull/5750)
```

Comment:
isnt it the case that the costing changes do in fact change the default behavior/routes for bikes? i think that if thats the case we should add some kind of disclaimer here in the changelog that routes will change. previously i would be against such a thing but ive softened on this over the years haha

Reply:
i guess technically an unset `use_roads` or a value of `.5` will not create different routes but any other option would potentially right? i think its worth saying to expect different routes

Reply:
@nilsnolde did you have some time to run routes in your area